test/methods.jl

```diff
@@ -145,22 +145,70 @@ end
     ti = Ti(1:4)
     da = DimArray(a, (y, ti))
     ys = (1, Y, Y(), :Y, y)
+    ys2 = (ys..., map(tuple, ys)...)
```

Comment:
Might help to add a few comments for what these test cases are for, and names for the test sets.